### ISSUE TITLE
Engine launch parameters not being picked up from a session file.

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1032]
When you go to create a movie with the movie wizard, one of the options is to have it give you the command to create the movie so that you can run it later. When it does that it saves a session file and gives you the command to run. One of the things that it saves in the session file is the launch parameters for launching the engine the same way you were when you went to save the movie. For example, if you were running in parallel with 4 processors it should run with 4 processors when you later create the movie. Instead it just runs a serial engine.

To reproduce this problem I did the following:

1) Open wave.silo.
2) When the engine launching window comes up, run with 4 processors.
3) Create a pseudocolor plot of pressure.
4) Bring up the movie wizard, have it create a movie of png images.
5) In the very last window, have it give you the command to run to save the movie and save the command.
6) Exit VisIt.
7) Execute the command given in step 5.

You will see that it starts a serial engine.

Here is the output I got, which clearly shows I ran it in parallel and then when I created the movie it ran a serial engine.

prism0{brugger}27: visit -v 2.0
Running: gui2.0.1
Running: viewer2.0.1 -host 127.0.0.1 -port 5600 -geometry 1512x1148+408+0
-borders 21,8,4,4 -shift 4,21 -preshift 0,0 -defer
Running: mdserver2.0.1 -host 127.0.0.1 -port 5601
Running: msub -v
LIBPATH=/usr/gapps/visit/2.0.1/linux-x86_64/lib,LD_LIBRARY_PATH=/usr/gapps/visit/2.0.1/linux-x86_64/lib:/usr/local/tools/mvapich-gnu/lib/shared:/usr/local/lib,VISITHOME=/usr/gapps/visit/2.0.1,VISITARCHHOME=/usr/gapps/visit/2.0.1/linux-x86_64,VISITPLUGINDIR=:/g/g17/brugger/.visit/linux-x86_64/plugins:/usr/gapps/visit/2.0.1/linux-x86_64/plugins
-l nodes=2:ppn=2 -l walltime=30:00 -q pbatch -A bdivp cd
/g/g17/brugger/test_engine_relaunch
 ulimit -c 0
    srun -n 4 "/usr/gapps/visit/2.0.1/linux-x86_64/bin/engine_par -host
prism0.llnl.gov -port 5600 -dir /usr/gapps/visit -noloopback -forcestatic
-timeout 480 -key 9dc351ccef252099c917
"

44529
prism0{brugger}28: /usr/gapps/visit/bin/visit -movie -v 2.0.1 -format png
-geometry 584x353 -output /g/g17/brugger/test_engine_relaunch/movie/movie -fps
10 -start 0 -end 70 -sessionfile
/g/g17/brugger/test_engine_relaunch/movie/movie.session
Running: cli2.0.1 -forceversion 2.0.1 -format png -geometry 584x353 -output
/g/g17/brugger/test_engine_relaunch/movie/movie -fps 10 -start 0 -end 70
-sessionfile /g/g17/brugger/test_engine_relaunch/movie/movie.session -s
/usr/gapps/visit/2.0.1/linux-x86_64/bin/makemoviemain.py -nowin
Running: viewer2.0.1 -host 127.0.0.1 -port 5600 -noint -geometry 584x353
-start 0 -end 70 -nowin -forceversion 2.0.1
/usr/gapps/visit/2.0.1/linux-x86_64/bin/makemoviemain.py:39:
DeprecationWarning: The xmllib module is obsolete.  Use xml.sax instead.
  #
Running: mdserver2.0.1 -host 127.0.0.1 -port 5600 -start 0 -end 70
-forceversion 2.0.1
VisIt: Message - VisIt imported a session from:
/g/g17/brugger/test_engine_relaunch/movie/movie.session.
Running: engine_ser2.0.1 -host 127.0.0.1 -port 5600 -dir /usr/gapps/visit
-timeout 480 -start 0 -end 70 -forceversion 2.0.1
VisIt: Message - Rendering window 1...
VisIt: Message - Saving window 1...
VisIt: Message - Saved /g/g17/brugger/test_engine_relaunch/movie/movie0000.png
VisIt: Message - Rendering window 1...
VisIt: Message - Saving window 1...

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 06/29/2010 02:42 pm
Original update: 07/13/2010 01:49 pm
Ticket number: 185
